client/main/index.tsx

```diff
@@ -0,0 +1,169 @@
+import Link from 'next/link';
+import Carousel from 'components/main/Carousel';
+import NewReview from 'client/main/NewReview';
+import NewRoom from 'client/main/NewRoom';
+import TabBar from 'components/main/TabBar';
+
+function Home() {
+  let userID;
+  let userNickname;
+  if (typeof window !== 'undefined') {
+    userID = localStorage.getItem('userId');
+    userNickname = localStorage.getItem('userNickname');
+  }
+
+  const searchClickHandler = () => {
+    sessionStorage.clear();
+    sessionStorage.setItem('search', 'true');
+  }
+
+  return (
+    <div className="bg-background_beige h-screen sm:w-[375px] sm:m-auto">
+      <TabBar />
+      <div className="pb-12 w-full">
+        {/* *학교필터 */}
+        <div className="flex items-center border-b-2 border-border_color text-3xl justify-between py-8 px-[18px]">
+          <div className="flex items-center justify-between">
+            <div className="px-4 Pretendard-Regular">한양대 ERICA</div>
+            {userID ? (
+              <div className="text-xl">{userNickname}님 안녕</div>
+            ) : (
+              <div className="text-xl">로그인하셈</div>
+            )}
+          </div>
+          <div className="flex Pretendard-Regular items-center justify-between">
+            <Link href="/login">
+              <a>
+                <div className="text-xl">간편로그인</div>
+              </a>
+            </Link>
+            <Image src={require('assets/icon/더보기.svg')} alt="더보기" />
+          </div>
+        </div>
+        {/* *로고와 메인글 */}
+        <header className="flex items-center text-4xl justify-between pt-8 pb-3 mx-[18px] relative">
+          <div>
+            <div className="sm:relative flex items-end justify-between">
+              <div className="w-[215px]">
+                <Image
+                  width={215}
+                  height={157}
+                  alt="메인캐릭터"
+                  src={require('assets/icon/main/main_char.png')}
+                />
+              </div>
+              <div className="absolute sm:left-[190px] sm:top-[85px] right-[35px] top-[110px] text-[23px] w-[140px] mb-5 Pretendard-SemiBold">
+                어떤 방을
+                <br />
+                찾고 계신가요?
+              </div>
+            </div>
+          </div>
+        </header>
+
+        {/* *지역검색  */}
+        <div className="relative h-20 mx-[18px]">
+          <div className="absolute  left-6 bottom-[0] flex items-center h-full">
+            <Image
+              alt="돋보기"
+              width={22}
+              height={22}
+              src={require('assets/icon/main/메뉴바_검색_활성화.svg')}
+            />
+          </div>
+          <Link href="/map">
+            <a>
+              <input
+                type="text"
+                className="bg-white rounded-[8rem] h-full text-xl pl-20 placeholder-font_border_color outline-0 w-full"
+                placeholder="지역, 지하철, 대학교 검색"
+                onClick={searchClickHandler}
+              ></input>
```

Comment:
```suggestion
 <input
                type="text"
                className="bg-white rounded-[8rem] h-full text-xl pl-20 placeholder-font_border_color outline-0 w-full"
                placeholder="지역, 지하철, 대학교 검색"
                onClick={searchClickHandler}
              />
```
처럼 self closing tag가 더 깔끔할 것 같아요